JavaScript/Node.js/sample.js

```diff
@@ -0,0 +1,120 @@
+import { BatchServiceClient, BatchSharedKeyCredentials } from "@azure/batch";
```

Comment:
Might want to switch this to require() to support older versions of Node.js

Reply:
I'd like to leave it as is so that it runs independent of Node entirely.